Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:8.10.2-jdk17 as build
+
+WORKDIR /app
+
+COPY . .
+
+RUN gradle bootjar
+
+FROM openjdk:17
+WORKDIR /app
+COPY --from=build /app/build/libs/argos-v1.jar .
+ENTRYPOINT ["java","-jar","argos-v1.jar"]
```

Comment:
When referring to the name of the jar, please use [globbing](https://www.baeldung.com/linux/bash-globbing). This is to avoid naming issues, if the name of the jar changes.

Reply:
done